src/css/popup.css

```diff
@@ -783,8 +792,20 @@ span ~ .panel-header-text {
   padding-inline-start: 16px;
 }
 
+/**
+ Bugfix: incorrect scrollbars/spacing when editing container with many assignments
+ https://github.com/mozilla/multi-account-containers/issues/948
+ */
+.edit-container-panel .columns {
+  overflow: hidden;
+}
+
 #edit-sites-assigned {
-  flex: 1;
+  /**
+   Bugfix: incorrect scrollbars/spacing when editing container with many assignments
+   https://github.com/mozilla/multi-account-containers/issues/948
```

Comment:
I think these code comments are a little excessive, I would remove them.

Reply:
Ok sure, I'll remove the comments